### ISSUE TITLE
allow to track commands in ruby scripts

### ISSUE THREAD

**Issue by soenkehahn** [Originate/check-protocols#81]
Currently in ruby scripts like the one below, no commands are being detected.

``` ruby
#!/usr/bin/env ruby
`touch foo`
```
